mease_lab_to_nwb/convert_syntalos/syntalosnwbconverter.py

```diff
@@ -19,22 +55,66 @@ class SyntalosNWBConverter(NWBConverter):
     )
 
     def get_metadata(self):
-        """Auto-populate as much metadata as possible."""
```

Comment:
why delete the docstring?

Reply:
Same reason as above, but I just realized the `NWBConverter .run_conversion` does indeed override a few things so I'm adding one there.